contracts/AuctionSuperApp.sol

```diff
@@ -307,6 +307,16 @@ contract AuctionSuperApp is SuperAppBase, AccessControlEnumerable, Pausable {
         return bid.contributionRate;
     }
 
+    /**
+     * @notice get the start time of the reclaimer bid to calculate reclaimer.claimPrice
+     * @param id License id
+     * @return reclaim start time
+     */
+    function getReclaimerStartTime(uint256 id) external view returns (uint256) {
```

Comment:
I think this should be the latest of `lastUserDeletion[owner]` and `currentOwnerBid[id].timestamp`.

- `currentOwnerBid[id].timestamp` will be updated if an owner lowers their payment to 0 but still has other parcels
- `lastUserDeletion[owner]` will be updated if an owner deletes their flow, which lowers their payment to 0 for all parcels they own

In either case, reclaiming will only be possible if `ownerBidContributionRate() == 0`. The super app checks for this before calling the reclaimer.